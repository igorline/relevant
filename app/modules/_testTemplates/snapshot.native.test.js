import React from 'react';
import renderer from 'react-test-renderer';
import AvatarBox from 'modules/user/mobile/avatar.component';

const user = { _id: 123, handle: 'handle', name: 'Name', relevance: { pagerank: 10 } };
const postTime = '1 day ago';

const props = {
  user,
  relevance: false,
  postTime,
  setSelected: jest.fn()
};

test('Snapshot AvatarBox Styled', () => {
  const tree = renderer.create(<AvatarBox {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});
