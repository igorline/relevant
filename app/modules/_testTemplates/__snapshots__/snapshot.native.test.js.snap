// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot AvatarBox Styled 1`] = `
<div
  className="css-view-1dbjc4n"
  data-focusable={true}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "WebkitAlignItems": "center",
      "WebkitBoxAlign": "center",
      "WebkitBoxDirection": "normal",
      "WebkitBoxFlex": 1,
      "WebkitBoxOrient": "horizontal",
      "WebkitBoxPack": "start",
      "WebkitFlexBasis": "0%",
      "WebkitFlexDirection": "row",
      "WebkitFlexGrow": 1,
      "WebkitFlexShrink": 1,
      "WebkitJustifyContent": "flex-start",
      "alignItems": "center",
      "cursor": "pointer",
      "flexBasis": "0%",
      "flexDirection": "row",
      "flexGrow": 1,
      "flexShrink": 1,
      "justifyContent": "flex-start",
      "msFlexAlign": "center",
      "msFlexDirection": "row",
      "msFlexNegative": 1,
      "msFlexPack": "start",
      "msFlexPositive": 1,
      "msFlexPreferredSize": "0%",
      "msTouchAction": "manipulation",
      "touchAction": "manipulation",
    }
  }
  tabIndex="0"
>
  <div
    className="css-view-1dbjc4n"
    style={
      Object {
        "WebkitFlexBasis": "auto",
        "borderBottomLeftRadius": "13px",
        "borderBottomRightRadius": "13px",
        "borderTopLeftRadius": "13px",
        "borderTopRightRadius": "13px",
        "flexBasis": "auto",
        "height": "26px",
        "marginRight": "5px",
        "msFlexPreferredSize": "auto",
        "overflowX": "hidden",
        "overflowY": "hidden",
        "width": "26px",
        "zIndex": 0,
      }
    }
  >
    <div
      className="css-view-1dbjc4n"
      style={
        Object {
          "backgroundColor": "rgba(0,0,0,0.00)",
          "backgroundImage": "url(\\"default_user.jpg\\")",
          "backgroundPosition": "center",
          "backgroundRepeat": "no-repeat",
          "backgroundSize": "cover",
          "bottom": "0px",
          "height": "100%",
          "left": "0px",
          "position": "absolute",
          "right": "0px",
          "top": "0px",
          "width": "100%",
          "zIndex": -1,
        }
      }
    />
    <img
      alt=""
      className="css-accessibilityImage-9pa8cd"
      draggable={false}
      src="default_user.jpg"
    />
  </div>
  <div
    className="css-view-1dbjc4n"
  >
    <div
      className="css-view-1dbjc4n"
      style={
        Object {
          "WebkitAlignItems": "flex-end",
          "WebkitBoxAlign": "end",
          "WebkitBoxDirection": "normal",
          "WebkitBoxOrient": "horizontal",
          "WebkitBoxPack": "start",
          "WebkitFlexDirection": "row",
          "WebkitJustifyContent": "flex-start",
          "alignItems": "flex-end",
          "flexDirection": "row",
          "justifyContent": "flex-start",
          "marginBottom": "2px",
          "msFlexAlign": "end",
          "msFlexDirection": "row",
          "msFlexPack": "start",
        }
      }
    >
      <div
        className="css-text-901oao"
        dir="auto"
        style={
          Object {
            "color": "rgba(36,36,37,1.00)",
            "fontFamily": "BebasNeueRelevantRegular",
            "fontSize": "17px",
            "fontWeight": "normal",
            "letterSpacing": "0.5px",
            "lineHeight": "17px",
            "marginBottom": "-2px",
          }
        }
      >
        Name
         
      </div>
    </div>
    <div
      className="css-text-901oao"
      dir="auto"
      style={
        Object {
          "color": "rgba(156,156,156,1.00)",
          "fontSize": "10px",
          "lineHeight": "10px",
        }
      }
    >
      @123
       
      1 day ago
    </div>
  </div>
</div>
`;
